src/backend/api/reservations.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const { on } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+
+router.get("/", async (request, response) => {
+    const reservations = await knex("reservations");
+    response.json(reservations);
+})
+
+router.post("/", async (request, response) => {
+
```

Comment:
would recommend to use a try catch block where you capture the error in case the insert fails and return an response to the client as an error code